test/12-assets.test.js

```diff
@@ -62,7 +62,8 @@ test('staged manifest assets fetched by run', async (t) => {
     Pear.app.dir = base
   })
   const run = await Helper.run({ link: appWithAssetsDir })
-  await Helper.untilResult(run.pipe)
+  const result = await Helper.untilResult(run.pipe)
+  t.is(result, 'hello')
```

Comment:
Am explicitly checking this now to make sure that running without the `--preflight` tag here behaves differently than running with `--preflight`